test/test_unary_ufuncs.py

```diff
@@ -1574,6 +1639,25 @@ def test_atanh_domain_float(self, device, dtype):
         self.assertEqual(torch.isinf(torch.atanh(sample)), inf_mask)
         self.assertEqual(torch.isinf(sample.atanh()), inf_mask)
 
+    @unittest.skipIf(TEST_ROCM, "skip on ROCM")
+    @dtypes(*torch.testing.get_all_dtypes(include_complex=False, include_bfloat16=False))
+    def test_frexp(self, device, dtype):
```

Comment:
Do we actually need this test if we have the OpInfo with a NumPy reference that will cross-check the results?

Reply:
Thanks for pointing this out, I would remove this redundant test in the next update.